### PR TITLE
register/login

### PR DESCRIPTION
**Pull request by tasnhubaislam**

### CODE REVIEW

**Review by farzanaisrat — APPROVED** [review]
Good!

**Review by yellow-tulips — APPROVED** [review]
Very beautiful!

**Review by joerossi215 — APPROVED** [review]
Looks good!

File: login.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <link rel="stylesheet" href="style.css">
+   <!-- Font Awesome Cdn Link -->
+   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css"/>
```
Comment (Taraque-A) [inline login.html:7]: Looks very nice.

File: login.html [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+</head>
+<body>
+  <div class="wrapper">
+    <h1>Hello Again!</h1>
+    <p>Welcome back you've <br> been missed!</p>
+    <form>
+      <input type="text" placeholder="Enter username">
+      <input type="password" placeholder="Password">
```
Comment (Taraque-A) [inline login.html:15]: Its look pretty.

File: register.html [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <link rel="stylesheet" href="style.css">
+   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css"/>
+</head>
+<body>
+  <div class="wrapper">
+    <h1>Sign up!</h1>
+    <p>Welcome to your Helping Hand <br> You been missing out!</p>
```
Comment (Taraque-A) [inline register.html:11]: Looks good.

**Review by Taraque-A — APPROVED** [review]

**Pull request merged**